editing.md

```diff
@@ -99,18 +99,110 @@ Which brings up:
 
 There are a number of configurable options here:
 
-- **CRAN Mirror** controls which CRAN mirror to download packages from by default.
+### Debugging
 
-- **R Engine** is set to the directory of your R installation. If you change
-  this setting, please restart Visual Studio.
+**Evaluate active bindings** Ensures that you always get the most up to date
+value when inspecting variables and properties in the Variable Explorer, and
+Debugger Tool Windows like Watch and Locals. The risk is that evaluating the
+expressions may cause side-effects, depending on how they were implemented.
 
-- **Send to R Console shortcut** lets the user redefine what keyboard shortcut
-sends the current editor selection to the REPL.
+**Show dot-prefixed variables** Shows variables prefixed with "." in Variable
+Explorer and Debugger Tool Windows.
 
-- **Load workspace when project opens**: when you open a RTVS project in VS if
-this option is set to *Yes*, it will automatically load the Workspace file
-(.RData) in the project directory. *Ask* will prompt the user.
+### General
 
-- **Save workspace when the project closes** will automatically write the R
-execution context to the .RData file when the project closes. Other options
-are *No* and *Ask*.
+**Survey/News Check** Survey/News is a feature that pushes news about RTVS
+updates and surveys. This setting controls the frequency of how often RTVS
+checks for updates.
+
+### Help
+
+**F1 Web browser** Controls how help is displayed when you are searching for a
+term using CTRL+F1. When set to `Internal`, you will see help rendered within a
+tool window in Visual Studio. When set to `External`, you will see help rendered
+using your default web browser.
+
+**F1 Web Search String** Controls how search terms are passed to your search
+engine when you press CTRL+F1 on a term in the editor. By default the string is
+`R site:stackoverflow.com`, which replaces `R` with your search term. The
```

Comment:
Actually it does not replace 'R'. 'R' helps to find results related to R rather than general search. So The query is 'term R  site:stackoverflow.com'